src/spdx/parser/rdf/relationship_parser.py

```diff
@@ -49,3 +49,25 @@ def parse_relationship(
     )
 
     return relationship
+
+
+def parse_implicit_relationship(
+    related_spdx_element_node: URIRef,
+    graph: Graph,
+    parent_node: URIRef,
+    doc_namespace: str,
+    relationship_type: RelationshipType,
```

Comment:
I find the argument order here confusing, as the "normal" order would be 
`parent_node`, `relationship_type`, `related_spdx_element_node`.
I also took some time to understand that `parent_node` refers to the SPDX element on the left side of the relationship, but I'm currently at a loss for a better name...

Reply:
Alright, I will change the order. In the first place, I thought I use the same for-loop as for the relationships in `rdf_parser.py` which would have needed this order but as this isn't possible, I can rearrange this.